### PR TITLE
Scorer

### PR DESCRIPTION
**Pull request by Izayuh**
Simple prototype for our scoring function.

### CODE REVIEW

**Review by tvu212 — APPROVED** [review]

**Review by Chris-Pyles — APPROVED** [review]

**Review by tvu212 — APPROVED** [review]

File: main.gs [hunk 1]
```diff
@@ -31,3 +31,44 @@ function matchGirls() {
...
+            || (outputData[i][4] == responseData[i][COLUMN_PREFERENCE_4])) {
+            score += 10;
+        }if((outputData[i][2] == responseData[i][COLUMN_PREFERENCE_5])
+            || (outputData[i][3] == responseData[i][COLUMN_PREFERENCE_5])
+            || (outputData[i][4] == responseData[i][COLUMN_PREFERENCE_5])) {
+            score += 11;
+        }if((outputData[i][2] == responseData[i][COLUMN_PREFERENCE_6])
+            || (outputData[i][3] == responseData[i][COLUMN_PREFERENCE_6])
+            || (outputData[i][4] == responseData[i][COLUMN_PREFERENCE_6])) {
+            score += 12;
```
Comment (beszel) [inline main.gs]: Write constants to remove these magic numbers. Can you refactor this block to reduce repetition? What about a helper function?

File: main.gs [hunk 2]
```diff
@@ -31,3 +31,44 @@ function matchGirls() {
         outputSheet.appendRow([firstName, lastName, preference_1, preference_2, preference_3]);
     }
 }
+
```
Comment (beszel) [inline main.gs]: Add doctag

**Review by beszel — CHANGES_REQUESTED** [review]

**Comment by beszel** [comment]
If a student doesn't get any of their preferences, they get a perfect score of 0. They should get a worse score than that.

**Review by Chris-Pyles — APPROVED** [review]

**Review by Chris-Pyles — APPROVED** [review]

**Comment by Chris-Pyles** [comment]
this SHOULD be set for approval now, but it says there are conflicts. I feel like this has to do with the fact that master branch has main.js and scorer has main.gs. We might want to make them the same filetype for the future.

**Pull request merged**